includes/Core/Assets/Assets.php

```diff
@@ -204,74 +196,43 @@ public function enqueue_asset( $handle ) {
 	 * Enqueues Google fonts.
 	 *
 	 * @since 1.0.0
+	 * @deprecated  n.e.x.t This method is no longer used as fonts are loaded as a normal style dependency now.
 	 */
 	public function enqueue_fonts() {
-		if ( $this->fonts_enqueued ) {
-			return;
+		_deprecated_function( __METHOD__, 'n.e.x.t' );
+
+		$assets = $this->get_assets();
+
+		if ( ! empty( $assets['googlesitekit-fonts'] ) && $assets['googlesitekit-fonts'] instanceof Asset ) {
+			$assets['googlesitekit-fonts']->enqueue();
 		}
+	}
 
+	/**
+	 * Get Google fonts src for CSS.
+	 *
+	 * @since n.e.x.t
+	 * @return false|string String URL src, or `false` if no font families to load.
```

Comment:
Nit-pick (that would be caught by #924 😆):
```suggestion
	 * @since n.e.x.t
	 *
	 * @return false|string String URL src, or `false` if no font families to load.
```

Also, maybe use `bool|string`? I've seen `false` in other places, but it's a bit strange since it's not a type.

Reply:
Right, I think maybe the IDE generated it 🤔  Will update 👍

Reply:
Now just a string per the other comment 😄